ipstack/help.md

```diff
@@ -17,9 +17,19 @@
 
 The connection configuration accepts the following parameters:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|cred_token|credential_token|None|True|API Token|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|cred_token|credential_token|None|True|API Token|None|e6541d-7f855e-eac9a1478afeb48fcbc4|
+
+Example input:
+
+```
+{
+  “domain”: “example.com”,
```

Comment:
This domain property is actually not used at all in this plugin - it's a consequence of using one of the older credential types. Can you update the credential type to: `credential_secret_key`?